### PR TITLE
Feature: Updated GlobalHeader to updated designs

### PR DESCRIPTION
**Pull request by andrewchough**
# Describe your PR

Related to https://github.com/Rebuild-Black-Business/RBB-Website/issues/59

## Pages/Interfaces that will change
- Every page will have the updated global header

### Screenshots / video of changes
**Desktop**
Screen Shot 2020-06-08 at 8 03 00 PM

**Mobile**
Screen Shot 2020-06-08 at 8 03 13 PM

**Dummy modal when Subscribe button is clicked**
Screen Shot 2020-06-08 at 8 10 33 PM

### CODE REVIEW

File: src/components/PrimaryNav.js [hunk 1]
```diff
@@ -15,12 +26,15 @@ const PrimaryNav = forwardRef(
...
     const theme = useTheme();
     const toUpperCase = text => text.toUpperCase();
+    const { isOpen, onOpen, onClose } = useDisclosure();
 
     // Layout effect prevents a flash of visibility when resizing the screen
     useLayoutEffect(() => {
       setIsVisible(isMedium);
     }, [isMedium]);
 
+    console.log(isVisible);
```
Comment (magnificode) [inline src/components/PrimaryNav.js:36]: Couple of leftover console logs here :)

Comment (andrewchough) [reply src/components/PrimaryNav.js:36]: @magnificode great catch 😅

**Comment by magnificode** [comment]
@andrewchough just found another strange thing. It looks like when looking at the nav after a page refresh, the menu links are misaligned. But after resizing the browser window they show up properly 🤔 

https://gyazo.com/b4436059605f410863e56158c3331c2b

**Comment by andrewchough** [comment]
> @andrewchough just found another strange thing. It looks like when looking at the nav after a page refresh, the menu links are misaligned. But after resizing the browser window they show up properly 🤔
> 
> https://gyazo.com/b4436059605f410863e56158c3331c2b

I'm seeing that too.. I'll take a look and see where the problem is

**Comment by andrewchough** [comment]
@magnificode Does everything look alright to you now?

**Comment by magnificode** [comment]
@andrewchough looking _excellent_ - thank you so much!

**Pull request merged**